torch_xla/core/xla_model.py

```diff
@@ -529,8 +529,8 @@ def all_reduce(reduce_type, inputs, scale=1.0, groups=None, cctx=None):
   """Performs an inplace reduce operation on the input tensor(s).
 
   Args:
-    reduce_type (string): One of ``REDUCE_SUM``, ``REDUCE_MUL``, ``REDUCE_AND``,
-      ``REDUCE_OR``, ``REDUCE_MIN`` and ``REDUCE_MIN``.
+    reduce_type (string): One of ``xm.REDUCE_SUM``, ``xm.REDUCE_MUL``,
+    ``xm.REDUCE_AND``, ``xm.REDUCE_OR``, ``xm.REDUCE_MIN`` and ``xm.REDUCE_MIN``.
```

Comment:
MIN appears here twice, MAX is missing.

Reply:
As was in original text, you are good reviewer, thank you. let me correct that.